app/[locale]/job/FormPayment.tsx

```diff
@@ -23,12 +25,34 @@ export type FormPaymentData = {
 export default function FormPayment(params: FormPaymentProps) {
   const { t } = useTranslation()
 
+  let formatDate = () => {
```

Comment:
there is a world where the date methods i wrote for the tests could be generalized as utils. however, it's important to have some separation of the code under test and the code used _to test_. i've left these separate for that reason, but this could be a change down the line. the deciding question there will be "by combining date utils, am i still testing what i think i'm testing?" [truss, maintainer of React USWDS disagrees with me](https://github.com/trussworks/react-uswds/blob/main/src/components/forms/DatePicker/utils.tsx)